### PR TITLE
[expo-updates][5/n] Remove classic updates

### PR DESCRIPTION
**Pull request by wschurman**
# Why

This removes the SDK version config from updates configuration since it is no longer used (it was only used by classic updates and expo go, and expo go didn't really use it since it overrides the selection policy).

# How

Remove, propagate outwards, grepping for things that were removed.

# Test Plan

Run all tests and build and run Expo Go.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by wschurman** [comment]
All tests that are failing are failing in main.

File: packages/@expo/cli/e2e/fixtures/with-blank/app.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "expo": {
+        "runtimeVersion": "1.0",
```
Comment (wschurman) [inline packages/@expo/cli/e2e/fixtures/with-blank/app.json:3]: @EvanBacon - this is the part of this PR that would be good for you to review. The template has runtime version in it, so this test now requires runtimeVersion in the fixture. (can no longer use expo-updates without specifying runtime version)

**Review by ide — APPROVED** [review]

File: packages/@expo/config-plugins/src/ios/IosConfig.types.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export type ExpoPlist = {
   EXUpdatesReleaseChannel?: string;
   EXUpdatesRuntimeVersion?: string;
   EXUpdatesRequestHeaders?: Record<string, string>;
-  EXUpdatesSDKVersion?: string;
+  EXUpdatesSDKVersion?: string; // removed, but kept in types so that it can be mutated (deleted) from existing plists
```
[the remark this answers is not in the document]
Comment (wschurman) [reply packages/@expo/config-plugins/src/ios/IosConfig.types.ts:46]: https://linear.app/expo/issue/ENG-10997/add-docblock

**Comment by wschurman** [comment]
Checks passed in stacked PRs, so they're a bit flakey when they fail in this PR. Landing.

**Pull request merged**